backend/uclapi/dashboard/migrations/0006_auto_20170516_2014.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
```

Comment:
Might need to do these migrations again based on what is on current master. If I understand migrations correctly?